volatility3/framework/symbols/windows/extensions/mft.py

```diff
@@ -27,6 +27,14 @@ class MFTAttribute(objects.StructType):
     """This represents an MFT ATTRIBUTE"""
 
     def get_resident_filename(self) -> str:
```

Comment:
This should probably have been returning `Optional[str]` all along.  Any chance we could fix that up please?